docs: update links in scrolling guide to English version

kickbelldev opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/lynx-family/lynx-website?pullRequest=80) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/lynx-family/lynx-website?pullRequest=80) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/lynx-family/lynx-website?pullRequest=80) it.</sub>

Huxpro reviewed (commented):
Thanks for working on this. Ideally...neither the en docs or zh docs should hardcode the language prefix as Rspress should be able to handle it correctly.

Could you help with removing both?

kickbelldev commented:
Thanks for the review! I've removed the hardcoded language prefixes (`en/` and `zh/`) from all internal documentation links.

Changes made:
- Removed `en/` prefix from English docs
- Removed `zh/` prefix from Chinese docs
- Converted relative paths to absolute paths for better maintainability

Huxpro reviewed (approved):
Thanks a lot!
